Add v5 PV 2 docs for iOS and Android

Sporiff opened the pull request:
Closes https://adjustcom.atlassian.net/browse/ADAPP-18204

Sporiff commented:
@shashanksu @Aditi3 I've added you as reviewers just to check everything looks right from an SDK perspective. Here are the direct links:

- [Android](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/android/features/events?version=v5#purchase-verification)
- [iOS](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/ios/features/events?version=v5#purchase-verification)

Aditi3 commented on src/content/docs/sdk/ios/features/events.mdx:
```diff
@@ -282,41 +282,134 @@ Parameters:
+let event = ADJEvent(eventToken: "g3mfiw")!
+event.setProductId("product-id")
+event.setTransactionId("transaction-id")
+event.setRevenue(0.01, currency: "EUR")
```
```suggestion
event.setReceipt(yourReceiptData)
```

Aditi3 commented on src/content/docs/sdk/ios/features/events.mdx:
````diff
@@ -282,41 +282,134 @@ Parameters:
+<Tab title="Objective-C" sync="objc">
+
+```objc
+ADJAppStorePurchase *appStorePurchase = [[ADJAppStorePurchase alloc] initWithTransactionId:yourTranscationId
+                                                                              productId:yourProductId
+                                                                              andReceipt:yourReceiptData];
````
```suggestion
ADJAppStorePurchase *appStorePurchase = [[ADJAppStorePurchase alloc] 
                                                                               initWithTransactionId:yourTranscationId
                                                                               productId:yourProductId
                                                                               andReceipt:yourReceiptData];
```

Aditi3 reviewed (approved):
I provided one extra parameter, which was not needed, and updated it with the right one, the rest looks good.

shashanksu commented:
@Sporiff Android part looks good to me.

Sporiff commented:
Following up from @uerceg 's comments on [ADAPP-18204](https://adjustcom.atlassian.net/browse/ADAPP-18204), I've moved the documentation for PV to a separate page for each platform. I've also updated the code snippets in line with the suggestion provided, and added the Unity documentation.

@Aditi3 @shashanksu could you just take another look at the updated code snippets? @uerceg Could you verify Unity?

- [Android](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/android/features/purchase-verification?version=v5)
- [iOS](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/ios/features/purchase-verification?version=v5)
- [Unity](https://dev-docs-git-adapp-18204-documentation-p-2966d1-product-content.vercel.app/en/sdk/unity/features/purchase-verification?version=v5)

Aditi3 reviewed (approved):
iOS part looks good! 🚀

shashanksu commented:
Android part looks good too!
